server/lib/src/entry.rs

```diff
@@ -492,6 +492,117 @@ impl Entry<EntryInit, EntryNew> {
     }
 }
 
+impl From<SchemaAttribute> for EntryInitNew {
+    fn from(value: SchemaAttribute) -> Self {
+        EntryInitNew::from(&value)
+    }
+}
+
+impl From<&SchemaAttribute> for EntryInitNew {
+    fn from(s: &SchemaAttribute) -> Self {
+        // Convert an Attribute to an entry ... make it good!
+        let uuid_v = vs_uuid![s.uuid];
+        let name_v = vs_iutf8![s.name.as_str()];
+        let desc_v = vs_utf8![s.description.to_owned()];
+
+        let multivalue_v = vs_bool![s.multivalue];
+        let sync_allowed_v = vs_bool![s.sync_allowed];
+        let replicated_v = vs_bool![s.replicated.into()];
+        let phantom_v = vs_bool![s.phantom];
+        let unique_v = vs_bool![s.unique];
+        let indexed_v = vs_bool![s.indexed];
+
+        let syntax_v = vs_syntax![s.syntax];
+
```

Comment:
Other than "it's prettier" is there a reason to do all these variable creations when they're just going to be used lower in the insert calls? Things this deep in the weeds tend to get called a lot...